### PR TITLE
New store, layer, style, filters, content.ftl for AUS_CHLA_DB

### PR DESCRIPTION
**Pull request by xhoenner**
To review once https://github.com/aodn/harvesters/pull/521 has been merged and deployed.

### CODE REVIEW

**Comment by xhoenner** [comment]
Harvester has run successfully on rc so this can now be tested @bpasquer.

File: workspaces/imos/JNDI_aus_chla_db/aus_chla_db_data/featuretype.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  </namespace>
+  <title>aus_chla_db_data</title>
+  <keywords>
+    <string>features</string>
+    <string>aus_chla_db_data</string>
+  </keywords>
+  <metadataLinks>
+    <metadataLink>
+      <type>text/xml</type>
+      <metadataType>other</metadataType>
```
Comment (bpasquer) [inline workspaces/imos/JNDI_aus_chla_db/aus_chla_db_data/featuretype.xml]: Why not  TC211  as usual?

File: workspaces/imos/JNDI_aus_chla_db/datastore.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <name>JNDI_aus_chla_db</name>
+  <type>PostGIS (JNDI)</type>
+  <enabled>true</enabled>
+  <workspace>
+    <id>WorkspaceInfoImpl-5f0a648d:1428d0d11a9:-8000</id>
+  </workspace>
+  <connectionParameters>
+    <entry key="dbtype">postgis</entry>
+    <entry key="encode functions">false</entry>
+    <entry key="jndiReferenceName">java:comp/env/jdbc/local_harvest_read</entry>
```
Comment (bpasquer) [inline workspaces/imos/JNDI_aus_chla_db/datastore.xml]: Change to point to harvest_read

**Comment by xhoenner** [comment]
changes made @bpasquer, thanks for the review

**Pull request merged**